### DISCUSSION TITLE
Use `node:alpine` docker image

### DISCUSSION THREAD

**Discussion by kinggoesgaming** [epicweb-dev/epic-stack#794] [Ideas]
In my testing, reduces the image and memory footprint significantly. The only change will be replacing `apt` commands with `apk` commands.

https://fly.io/blog/fly-with-alpine/

**Comment by kentcdodds** [comment]
Thanks for the suggestion, but I really don't think cutting the image size by ~25MB (~33%) is really significant enough to justify the downsides like things unexpectedly not working. Especially when you consider most users of the Epic Stack aren't familiar with linux distros at this level. Lots of hours of frustration when things aren't working because they didn't know alpine doesn't support something debian does.

**Reply by kinggoesgaming** [reply]
That a fair assessment. Linux is confusing as it is, giving a uncommon distro will definitely throw users off, not to mention the debugging and help nightmare.

Something (in the future), we could provide it as an option in `create-epic-app`, once we have done better testing.

Just throwing it out...

**Reply by kentcdodds** [reply]
One of the goals of the Epic Stack is to reduce the number of decisions developers need to make. There has to be a **really good reason** to ask the developer a question during initialization.

**Reply by jacobparis** [reply]
this could be a good candidate for a page in the docs explaining the cost/benefit of switching to alpine though, especially if it's easy to switch

**Reply by kentcdodds** [reply]
Yep. Definitely open to a docs PR!
